src/main.js

```diff
@@ -212,10 +207,14 @@ function showMyPoster() {
 //returnToMain()
 //mainPage should display new saved poster
 
-function saveCreatedPoster() {
-  console.log("are you here")
-  var newPoster = new Poster({imageURL: posterImgURLInput, title: posterTitleInput, quote: posterQuoteInput});
-  images.push(posterImgURLInput.value);
-  titles.push(posterTitleInput.value);
-  quotes.push(posterQuoteInput.value);
+var currentPoster;
```

Comment:
Just convention, but usually global variables are all set at the top of the file.